v2.2/load-based-splitting.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Load-Based Splitting 
+summary: To optimize your cluster's performance, CockroachDB can split frequently accessed keys into their own ranges.
+toc: true
+---
+
+<span class="version-tag">New in v2.2:</span> To optimize your cluster's performance, CockroachDB can split frequently accessed keys into smaller ranges. In conjunction with [load-based rebalancing](architecture/replication-layer.html#load-based-replica-rebalancing), load-based splitting distributes load evenly across your cluster.
+
+## Settings
+
+### Enable/disable load-based splitting
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.range_split.by_load_enabled` to:
+
+- `true` to enable load-based splitting _(default)_
+- `false` to disable load-based splitting
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET CLUSTER SETTING kv.range_split.by_load_enabled = true;
+~~~
+
+#### When to enable load-based splitting
+
+Load-based splitting is on by default and beneficial in almost all situations. 
+
+#### When to disable load-based splitting
+
+You might want to disable load-based splitting when troubleshooting range-related issues under the guidance of our developers.
+
+### Control load-based splitting threshold
+
+Use [`SET CLUSTER SETTING`](set-cluster-setting.html) to set `kv.range_split.load_qps_threshold` to the queries-per-second (QPS) at which you want to consider splitting a range (defaults to `250`):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET CLUSTER SETTING kv.range_split.load_qps_threshold = 200;
+~~~
+
+#### When to modify the load-based splitting threshold
+
+Some workloads might find splitting ranges more aggressively (i.e. a lower QPS threshold) can improve performance.
+
+On the other hand, some workloads with very large machines might want to increase the QPS threshold to split more conservatively.
+
+## How load-based splitting works
+
+Whenever a range exceeds the cluster's setting for `kv.range_split.load_qps_threshold`, the range becomes eligible for load-based splitting.
+
+At that point, we begin gathering per-key metrics to determine whether or not a split would benefit the cluster's performance based on the following metrics:
```

Comment:
> based on the following metrics
Further down the page you refer to these as " the balance factor and split crossing heuristics".  I think we should call them heuristics here as well.